piqueserver/player.py

```diff
@@ -265,8 +266,38 @@ def on_team_join(self, team: 'FeatureTeam') -> HookValue:
                     return False
                 self.send_chat('Team is full, moved to %s' % other_team.name)
                 return other_team
+
+        # Add the user to their relevant groups
+        self.user_types.add("all")
+
+        team_usertype = None
+
+        if team is self.protocol.team_1:
+            team_usertype = "team_1"
```

Comment:
Perhaps we could use `team1` and `team2` to be consistent with the toml config sections?

Reply:
That's a good point. Kind of unfortunate that our config and code differ now.